src/hb-ot-name-table.hh

```diff
@@ -100,7 +100,9 @@ struct NameRecord
 		    const void *dst_base) const
   {
     auto *out = c->embed (this);
-    out->offset = 0;
+    if (!out)
+      return nullptr;
+
```

Comment:
Instead of the three lines, just have one line:
```
if (unlikely (!out)) return_trace (false);
```
and yes, add tracing to the function please.